src/utils/notes.js

```diff
@@ -0,0 +1,27 @@
+const NOTES_MARKER = 'Notes: ';
```

Comment:
One thought is we may want this to be dynamic from the CLI or some config later. I'm not sure it's possible to override with this structure. Or we can say this is the API for md(x) notes and it's static :)